config/locales/in_person_proofing/en.yml

```diff
@@ -112,11 +115,15 @@ en:
               missing_month_day_year: Enter a date of birth
               range_min_age: You must be over 13 years of age to use %{app_name}
               range_overflow: Enter a date that is in the past
+        same_address_as_id: Is your current residential address listed on your state-issued ID?
+        same_address_as_id_no: No, my current residential address is not listed on my state-issued ID
+        same_address_as_id_yes: Yes, my current residential address is listed on my state-issued ID
         state_id_jurisdiction: State
         state_id_jurisdiction_hint: Select the state shown on your ID
         state_id_jurisdiction_prompt: '- Select -'
         state_id_number: ID number
         state_id_number_hint: May include letters and numbers
+        zipcode: ZIP Code
     headings:
       address: Enter your current address
```

Comment:
looking at the figma i think address might need to be updated to "Enter your current residential address".

Reply:
Hi, changes to the address page are not in scope for this PR.